lib/connect-dynamodb.js

```diff
@@ -106,7 +106,7 @@ module.exports = function (connect) {
     DynamoDBStore.prototype.get = function (sid, fn) {
 
         sid = this.prefix + sid;
-        var now = +new Date;
+        var now = Math.floor(Date.now() / 1000);
```

Comment:
@rafaelrpinto won't this be wrong for existing records in the table? Any recommendations for migrating existing tables?